src/connectedmachine/azext_connectedmachine/generated/action.py

```diff
@@ -7,4 +7,88 @@
 # Changes may cause incorrect behavior and will be lost if the code is
 # regenerated.
 # --------------------------------------------------------------------------
+
+
 # pylint: disable=protected-access
+
+# pylint: disable=no-self-use
+
+
+import argparse
+from collections import defaultdict
+from knack.util import CLIError
+
+
+class AddStatus(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        action = self.get_action(values, option_string)
+        namespace.status = action
+
+    def get_action(self, values, option_string):
+        try:
+            properties = defaultdict(list)
+            for (k, v) in (x.split('=', 1) for x in values):
+                properties[k].append(v)
+            properties = dict(properties)
+        except ValueError:
+            raise CLIError('usage error: {} [KEY=VALUE ...]'.format(option_string))
```

Comment:
Could you please use the specific error type `InvalidArgumentValueError` instead of `CLIError`?